.github/workflows/build_workflow.yml

```
@@ -105,14 +107,13 @@ jobs:
       - if: ${{ steps.skip_check.outputs.should_skip != 'true' }}
         name: Install mpas_tools
         run: |
-          conda create -y -n mpas_tools_dev -c ${CONDA_PREFIX}/conda-bld/ \
-              -c conda-forge mpas_tools python=${{ matrix.python-version }} \
-              sphinx sphinx-multiversion mock sphinx_rtd_theme
+          conda install -y -c ${CONDA_PREFIX}/conda-bld/ \
+            mpas_tools python=${{ matrix.python-version }} \
+            sphinx sphinx-multiversion mock sphinx_rtd_theme
```

This should be updated to match:
https://github.com/MPAS-Dev/MPAS-Tools/blob/master/conda_package/dev-spec.txt#L37-L39
```suggestion
            sphinx mock sphinx_rtd_theme
```